src/Symfony/Component/TypeInfo/TypeFactoryTrait.php

```diff
@@ -160,31 +160,31 @@ public static function collection(BuiltinType|ObjectType|GenericType $type, Type
     /**
      * @return CollectionType<BuiltinType<TypeIdentifier::ARRAY>>
      */
-    public static function array(Type $value = null, Type $key = null, bool $asList = false): CollectionType
+    public static function array(?Type $value = null, ?Type $key = null, bool $asList = false): CollectionType
     {
         return self::collection(self::builtin(TypeIdentifier::ARRAY), $value, $key, $asList);
     }
 
     /**
      * @return CollectionType<BuiltinType<TypeIdentifier::ITERABLE>>
      */
-    public static function iterable(Type $value = null, Type $key = null, bool $asList = false): CollectionType
+    public static function iterable(?Type $value = null, ?Type $key = null, bool $asList = false): CollectionType
     {
         return self::collection(self::builtin(TypeIdentifier::ITERABLE), $value, $key, $asList);
     }
 
     /**
      * @return CollectionType<BuiltinType<TypeIdentifier::ARRAY>>
      */
-    public static function list(Type $value = null): CollectionType
+    public static function list(?Type $value = null): CollectionType
```

Comment:
can you please submit all CS changes separately to keep the diff clean on this one?